.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+  "$schema": "http://json.schemastore.org/prettierrc",
+  "printWidth": 80,
```

Comment:
please use union config, and `printWidth` is unnecessary

Reply:
Does "union config"  means `@mjolnir/prettier-config` ?

Reply:
Of course, SD.